v2/lint/registration_test.go

```diff
@@ -145,33 +145,35 @@ func TestRegister(t *testing.T) {
 			},
 			registry:      dupeReg,
 			expectNames:   []string{"goodLint", egLint.Name},
-			expectSources: SourceList{MozillaRootStorePolicy, egLint.Source},
+			expectSources: SourceList{egLint.Source, MozillaRootStorePolicy},
 		},
 	}
 
 	for _, tc := range testCases {
-		var reg *registryImpl
-		if tc.registry == nil {
-			reg = NewRegistry()
-		} else {
-			reg = tc.registry
-		}
-
-		err := reg.register(tc.lint, tc.init)
-		if err == nil && tc.expectErr != nil {
-			t.Errorf("expected err %v, got nil", tc.expectErr)
-		} else if err != nil && err.Error() != tc.expectErr.Error() {
-			t.Errorf("expected err %v got %v", tc.expectErr, err)
-		} else if err == nil {
-			if !reflect.DeepEqual(reg.Names(), tc.expectNames) {
-				t.Errorf("expected names %v, got %v", tc.expectNames, reg.Names())
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
This diff is because I noticed I forgot to wrap these table tests in a `t.Run` when I wrote this test the first time. It made it hard to spot which testcases were causing a failure.

Reply:
The rest of the diffs are from sort order changes from the new source.